src/components/NavBar/NavBar.js

```diff
@@ -6,11 +6,13 @@ import '../../css/NavBar/NavBar.css'
 
 export default class NavBar extends React.Component {
   render() {
+    const loggedInUser = this.props.loggedInUser

```

Comment:
Can use destructuring here

Reply:
So what's the benefit of using destructing when there's only one item to pull out?

Reply:
just makes the code scalable in case we want more than one variable in the future.